lib/model.py

```diff
@@ -305,16 +305,21 @@ def runN(self, periods=DEF_TIME):
     def handle_womb(self):
         """
         This method adds new agents from the womb.
+        The womb should move up into model eventually.
         """
         self.env.handle_womb()
 
     def add_child(self, group):
         """
         Put a child agent in the womb.
         group: which group will add new agent
+        The womb should move up into model eventually.
         """
         self.env.add_child(group)
 
+    def get_pop_hist(self):
```

Comment:
If this is new there should be a test for this also?
Or is it already present?

Reply:
Yes, I will write the test... but I wanted to get Troy's endpoint working.